docs/dev/apis/csync.md

```diff
@@ -6,29 +6,52 @@ description: A guide to synchronizing a BepInEx config file between host and cli
 
 # CSync (Config Syncing Library)
 
+## About CSync
+
+CSync is a utility library for conveniently syncing the values of some BepInEx `ConfigEntry`
+instances from host to clients.
+
+::: details History
+
+CSync was originally authored by [@Owen3H](https://github.com/Owen3H). Sadly, Owen fell out of favour
+with the Lethal Company Modding discord and removed CSync from the Thunderstore.
+
+This led to the creation of a fork, under the [@lc-sigurd](https://github.com/lc-sigurd) organisation. As of
+April 2024, the fork has undergone extensive changes, resolving many issues that remain unfixed in Owen's
+distribution of CSync.
+
+Shortly after removing CSync from the Thunderstore in March, Owen reinstated the mod. It is nonetheless
+recommended to use Sigurd's distribution of CSync.
+
+For unknown reasons, Owen has transferred ownership of the [original CSync repository](https://github.com/DeathWrench/CSync) to
+[@DeathWrench](https://github.com/DeathWrench).
+
+:::
+
 ## Setup
-There are two ways to depend upon **CSync**, but I recommend using the [NuGet package](https://www.nuget.org/packages/Owen3H.BepInEx.CSync).<br>
-This will automatically include both an assembly reference and documentation as opposed to Thunderstore.
+There are two ways to depend upon **CSync**, but using the [NuGet package](https://www.nuget.org/packages/Owen3H.BepInEx.CSync)
```

Comment:
Is this supposed to link to Owen's dist, or to the Sigurd fork?

Reply:
😅